pkg/find/vulnerabilities/trivy/scanner.go

```diff
@@ -226,7 +231,9 @@ func (s *Scanner) GetVulnerabilityReportsByScanJob(ctx context.Context, job *bat
 		if err != nil {
 			return
 		}
-		reports[c.Name], err = s.converter.Convert(logReader)
+
+		imageRef := c.Args[len(c.Args)-1]
```

Comment:
It is kind of workaround because `job.Spec.Template.Spec.Containers` doesn't have the field representing the scanned image name.